lib/rack/insight/enable-button.rb

```diff
@@ -13,7 +13,7 @@ def call(env)
       @env = env
       status, headers, body = @app.call(@env)
 
-      if !body.nil? && !body.empty?
+      if !body.nil? && body.respond_to?('body') && !body.body.empty?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.